types/react-redux/index.d.ts

```diff
@@ -537,7 +551,7 @@ export function useDispatch<A extends Action = AnyAction>(): Dispatch<A>;
  *   return <div>{counter}</div>
  * }
  */
-export function useSelector<TState, TSelected>(
+export function useSelector<TState = SafeRootState, TSelected = any>(
```

Comment:
I'm not sure I agree with this change. By doing this the type of `TState` becomes `any` when no type parameters are given, while before this would result in `unknown` which seems more reasonable to me

Reply:
Good point, I'll fix it